provider/provider.go

```diff
@@ -431,13 +431,25 @@ func (p *ProviderCondition) Evaluate(ctx context.Context, log logr.Logger, condC
 		}
 		incidents = append(incidents, i)
 	}
+
+	// If there are no incidents, don't generate any violations
+	if len(incidents) == 0 && len(resp.Incidents)-len(incidents) > 0 {
+		log.V(5).Info("filtered out all incidents based on dep label selector", "filteredOutCount", len(resp.Incidents)-len(incidents))
```

Comment:
When is v(5) logged again? it is not in the default logs just if you increase verbosity right?

Reply:
I think it should be fine, don't worry about it. I think that in the future, we need guidelines for verbosity levels, but because we don't have any, this probably means it is a followup. Resolving the conversation and this LGTM